lib/Repository.js

```diff
@@ -496,21 +541,6 @@ class Repository extends Requestable {
 
    /**
     * Update a pull request
-    * @deprecated since version 2.4.0
-    * @see https://developer.github.com/v3/pulls/#update-a-pull-request
-    * @param {number|string} number - the number of the pull request to update
-    * @param {Object} options - the pull request description
-    * @param {Requestable.callback} [cb] - will receive the pull request information
-    * @return {Promise} - the promise for the http request
-    */
-   updatePullRequst(number, options, cb) {
```

Comment:
@clayreimann You were asking about a breaking change that would necessitate a major version bump. This would be it.